LoopMusic/Source/ViewControllers/LoopFinder/LoopFinderViewController.swift

```diff
@@ -59,9 +64,12 @@ class LoopFinderViewController: UIViewController, LoopScrubberContainer, UITextF
         } else {
             loopStartSeconds = 0
         }
-        loopStartField.text = NumberUtils.formatNumber(loopStartSeconds)
-        MusicPlayer.player.loopStartSeconds = loopStartSeconds
-        updateManualLoopTimes()
+        // Only do an update if the actual sample value will change
+        if MusicPlayer.player.convertSecondsToSamples(loopStartSeconds) != MusicPlayer.player.loopStart {
```

Comment:
This can use `MusicPlayer.player.loopStartSeconds` instead of manually converting seconds to samples. Same for `loopEndSeconds`.

Reply:
It shouldn't, actually. Converting to samples the double value to the closest sample, which is more appropriate when checking for changes.

Reply:
Oops, floating point equality. Good point.